src/main/java/com/google/edith/servlets/UserStatsServlet.java

```diff
@@ -33,7 +33,7 @@
 
```

Comment:
Can't comment line above, but why not make this var final as well?

Reply:
Done.